test/test_log_format.yml

```
@@ -0,0 +1,17 @@
+# GIVEN a webserver with VIRTUAL_HOST set to web1.tld
+web1:
+  image: web
+  expose:
+    - "81"
+  environment:
+    WEB_PORTS: 81
+    VIRTUAL_HOST: web1.tld
```

Client requests to names containing `nginx-proxy` will have their DNS lookups hijacked during testing to point to the nginx-proxy container:

```suggestion
    VIRTUAL_HOST: nginx-proxy.text
```